src/box/lua/feedback_daemon.lua

```diff
@@ -217,6 +217,54 @@ local function fill_in_schema_stats(features)
     features.schema = cached_schema_features
 end
 
+local function read_first_file(pattern)
+    local path = fio.glob(pattern)[1]
+    if not path then
+        local err = string.format(
+            'Tarantool repo not installed: nothing matches "%s"',
+            pattern
+        )
+        return nil, err
+    end
+
+    local fh, err = fio.open(path, {'O_RDONLY'})
+    if not fh then
+        return nil, err
+    end
+
+    local s, err = fh:read()
+    fh:close()
+    if type(s) ~= 'string' then
+        return nil, err
+    end
+
+    return s
+end
+
+local function extract_repo_url_apt()
+    local content, err = read_first_file('/etc/apt/sources.list.d/tarantool_*.list')
+    if not content then
+        return nil, err
+    end
+
+    return content:match('deb ([^ ]*)')
+end
+
+local function extract_repo_url_yum()
+    local content, err = read_first_file('/etc/yum.repos.d/tarantool_*.repo')
```

Comment:
Will miss a repo with dash, like `tarantool-2.5.repo`.

https://github.com/tarantool/doc/issues/1492#issuecomment-685740902

Reply:
Thanks, I've expanded wildcard to match `tarantool-2.5.repo` as well